### PR TITLE
🐛 Don't Increment Module Progress if it will exceed 100

### LINKED ISSUE #459: 🐛 Interactive Console Progress - Module appears to finish before actually finished

### PR DESCRIPTION
**Pull request by MattParkerDev**
Closes #459

### CODE REVIEW

**Comment by MattParkerDev** [comment]
I'm happy to change 95 to 99 or whatever you'd prefer :)

**Comment by thomhurst** [comment]
Nice one! Can you add a line to the ReleaseNotes.md in sec/ModularPipelines.Build?

It'll be useful when generating a new release.

File: src/ModularPipelines/Helpers/ProgressPrinter.cs [hunk 1]
```diff
@@ -173,7 +173,7 @@ private static string GetModuleExtraInformation(ModuleBase module)
                 var ticksPerSecond = 100 / totalEstimatedSeconds;
```
Comment (thomhurst) [inline src/ModularPipelines/Helpers/ProgressPrinter.cs:173]: Could you also turn this into 100.0? The fact it's an int is probably causing rounding issues which might be the reason for this.

Comment (MattParkerDev) [reply src/ModularPipelines/Helpers/ProgressPrinter.cs]: done!

**Comment by thomhurst** [comment]
Thanks again @MattParkerDev ! New version is out :smile:

**Pull request merged**